src/main/flight/mixer_tricopter.h

```diff
@@ -73,4 +73,82 @@ int16_t triGetMotorCorrection(uint8_t motorIndex);
  */
 _Bool triEnableServoUnarmed(void);
 
+#ifdef USE_SERVOS
```

Comment:
I was thinking along that path too, for a short while. I wanted to get a test running quickly so I dropped it.
Tried to follow the example from other units.
There is nothing dangerous about exposing type definitions, but variables and functions should be on a need-to-know basis (but without duplicating the same code lines).
Have to look again. Actually, this is what I was thinking of when I wrote "More cleaning might be good" in the PR comment.